webpack.config.devtools.js

```diff
@@ -8,7 +8,7 @@ webpackConfig.output.library = "Debugger";
 webpackConfig.plugins = [
   new webpack.DefinePlugin({
     "process.env": {
-      NODE_ENV: JSON.stringify("DEVTOOLS_PANEL"),
+      NODE_ENV: JSON.stringify("production"),
```

Comment:
changed this so that many libraries will work with the production mentality, we might also want a different flag to differentiate between the panel and the standalone app, but for now... should be fine